File: EWSWebDemoVB/MenuLayout.master.vb
Imports System.Configuration

Partial Class MenuLayout
    Inherits System.Web.UI.MasterPage

    Protected Overrides Sub OnLoad(ByVal e As System.EventArgs)
        MyBase.OnLoad(e)

        ' Get title from Web.Config
        If Not Page.IsPostBack Then
            lblAppName.Text = ConfigurationManager.AppSettings("MenuAppName")
        End If

    End Sub
End Class

File: EWSWebDemoCS/MenuLayout.master.cs
using System;
using System.Configuration;
using System.Web.UI;
using System.Web.UI.WebControls;

public partial class MenuLayout : System.Web.UI.MasterPage
{
    protected void Page_Load(object sender, EventArgs e)
    {
        // Get title from Web.Config
        if(!Page.IsPostBack)
            lblAppName.Text = ConfigurationManager.AppSettings["MenuAppName"];
    }
}
